test/fixtures/v5/continue/sharp_left_default.json

```diff
@@ -12,10 +12,13 @@
         "es": "Continua cerrada a la izquierda",
         "fr": "Continuer franchement à gauche",
         "id": "Terus tajam kiri",
+        "it": "Continua Leggera svolta a sinistra",
```

Comment:
There are problems with the Italian translation of "sharp ...", which does not sound very good at this time.
Could I have a (street?) example where this instruction should be used?